src/MIDebugEngine/Engine.Impl/ExceptionManager.cs

```diff
@@ -278,6 +278,27 @@ public void SetAllExceptions(enum_EXCEPTION_STATE dwState)
             }
         }
 
+        public int TryGetExceptionBreakpoint(string bkptno, out EXCEPTION_INFO exceptionInfo)
```

Comment:
You can make this function simpler by saving a `Dictionary<ulong, EXCEPTION_INFO>` and have this method `TryGetValue`.